api/src/main/java/edu/cornell/mannlib/vitro/webapp/edit/n3editing/configuration/generators/AddAttendeeRoleToPersonGenerator.java

```diff
@@ -8,82 +8,84 @@
-			"<http://purl.org/ontology/bibo/Interview> "+
-			"<http://vivoweb.org/ontology/core#Meeting> "+
-			"<http://purl.org/ontology/bibo/Performance> "+
-			"<http://vivoweb.org/ontology/core#Presentation> "+
-			"<http://vivoweb.org/ontology/core#InvitedTalk> "+
-			"<http://purl.org/ontology/bibo/Workshop> "+
-			"<http://vivoweb.org/ontology/core#EventSeries> "+
-			"<http://vivoweb.org/ontology/core#ConferenceSeries> "+
-			"<http://vivoweb.org/ontology/core#SeminarSeries> "+
-			"<http://vivoweb.org/ontology/core#WorkshopSeries> ";
-	boolean isShowRoleLabelField() {
-		return false;
-	}
+        // UQAM Replacing the above hard coding assignment by a dynamic assignment that takes into account the linguistic context
+        ConstantFieldOptions filedOptions = GeneratorUtil.buildConstantFieldOptions(vreq, DESCRIBE_QUERY);
+        return filedOptions;
+    }
+    /*
+     * UQAM get attributes for this specific subject
+     */
+    private static String DESCRIBE_QUERY = " describe "+
+            "<http://purl.org/NET/c4dm/event.owl#Event> "+
+            "<http://vivoweb.org/ontology/core#Competition> "+
+            "<http://purl.org/ontology/bibo/Conference> "+
+            "<http://vivoweb.org/ontology/core#Course> "+
+            "<http://vivoweb.org/ontology/core#Exhibit> "+
+            "<http://purl.org/ontology/bibo/Hearing> "+
+            "<http://purl.org/ontology/bibo/Interview> "+
+            "<http://vivoweb.org/ontology/core#Meeting> "+
+            "<http://purl.org/ontology/bibo/Performance> "+
+            "<http://vivoweb.org/ontology/core#Presentation> "+
+            "<http://vivoweb.org/ontology/core#InvitedTalk> "+
+            "<http://purl.org/ontology/bibo/Workshop> "+
+            "<http://vivoweb.org/ontology/core#EventSeries> "+
+            "<http://vivoweb.org/ontology/core#ConferenceSeries> "+
+            "<http://vivoweb.org/ontology/core#SeminarSeries> "+
+            "<http://vivoweb.org/ontology/core#WorkshopSeries> ";
+    boolean isShowRoleLabelField() {
+        return false;
+    }
 
-	/*
-	 * Use the methods below to change the date/time precision in the
-	 * custom form associated with this generator. When not used, the
-	 * precision will be YEAR. The other precisons are MONTH, DAY, HOUR,
-	 * MINUTE, TIME and NONE.
-	 */
-	/*
+       /*
+        * Use the methods below to change the date/time precision in the
+        * custom form associated with this generator. When not used, the
+        * precision will be YEAR. The other precisons are MONTH, DAY, HOUR,
+        * MINUTE, TIME and NONE.
+        */
+    /*
         public String getStartDatePrecision() {
             String precision = VitroVocabulary.Precision.MONTH.uri();
-    	    return precision;
+            return precision;
         }
 
         public String getEndDatePrecision() {
             String precision = VitroVocabulary.Precision.DAY.uri();
-    	    return precision;
+            return precision;
```

Comment:
It is only a minor difference, but I don't see this update in [Michel's original branch](https://github.com/vivo-project/VIVO/compare/sprint-i18n-whitespace?expand=1#diff-292a7b7c68e605ea463b9b9511cca2ceR86)